package.json

```diff
@@ -83,6 +86,8 @@
     "karma-sourcemap-loader": "^0.3.7",
     "karma-webpack": "^2.0.9",
     "mocha": "^4.1.0",
+    "node-sass": "^4.9.0",
```

Comment:
lots of new deps here... anyway we can async load this only for VAST?

Reply:
would be good to bundle this all in the maestro-videojs-vast player and async import it maybe

Reply:
why on earth do we need this? are we compiling SASS style files? O_o

Reply:
v1 was, fixing this